src/components/ShowPage.module.scss

```diff
@@ -36,3 +38,29 @@
     max-width: 30em;
   }
 }
+
+.headerWithShareBtn {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;  
+}
+
+.showNameHeader {
+  margin-right: 5px;
+}
+
+.shareButton {
+  border: none !important;
+  padding-top: 9px;
+}
+
+.shareIcon:hover {
+  opacity: 1 !important;
+  color: $mdc-theme-primary !important;
```

Comment:
Is it possible to not use `!important`? These are very costly in the long run. Since you already added so many we should just leave them, but we may be bitten by this in the future.